### PR TITLE
Install lustre-fs-operator and lustre-csi-driver directly from github

### PR DESCRIPTION
**Pull request by roehrich-hpe**
Use "kubectl apply -k", pointing to github, to install lustre-fs-operator and lustre-csi-driver.  The local copies of these repos are not referenced.

Extend the "Repository" config to allow the new information to be specified. To have nnf-deploy revert to the old way of handling these repos, change the 'useRemoteK' value to false for the given repo.

Add a wait phase for cert-manager, because our NNF stuff needs the cert-manager webhook to be live when they start up.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -175,10 +175,15 @@ type RepositoryConfigFile struct {
 }
 
 type Repository struct {
-	Name        string
-	Overlays    []string `yaml:",flow"`
-	Development string
-	Master      string
+	Name            string
+	Overlays        []string `yaml:",flow"`
```
Comment (bdevcich-hpe) [inline config/config.go:179]: Why is there no name in the `yaml:` here?

File: main.go [hunk 2]
```diff
@@ -499,26 +531,46 @@ func runKubectlLabelOrTaint(ctx *Context, node string, kctlCmd string, labelsOrT
...
+		fmt.Printf("\n\033[1mThe cluster is still waiting for cert-manager to start.\nPlease run `nnf-deploy init` after cert-manager is running.\033[0m\n\n")
 		return err
 	}
-
 	return nil
 }
 
+func installMPIOperator(ctx *Context) error {
+	fmt.Println("Installing mpi-operator...")
+	return runKubectlApplyF(ctx, "https://raw.githubusercontent.com/kubeflow/mpi-operator/v0.4.0/deploy/v2beta1/mpi-operator.yaml")
```
Comment (bdevcich-hpe) [inline main.go:571]: Would we want to store the path to the yamls for `mpi-operator` and `cert-manager` in repositories.yaml?

Comment (roehrich-hpe) [reply main.go:571]: That's tempting.  Maybe in the next PR.

File: config/config.go (hunk 1 above)
Comment (roehrich-hpe) [reply config/config.go:179]: I guess it's smart enough to get by without that and picks the field name.

**Review by bdevcich-hpe — APPROVED** [review]

**Review by matthew-richerson — APPROVED** [review]

**Pull request merged**